radix-transaction-scenarios/src/scenarios/account_locker.rs

```diff
@@ -207,15 +207,15 @@ impl ScenarioCreator for AccountLockerScenarioCreator {
                                 state.account_rejecting_fungible_resource.unwrap(),
                                 ACCOUNT_SET_RESOURCE_PREFERENCE_IDENT,
                                 AccountSetResourcePreferenceInput {
-                                    resource_address: state.fungible_resource.unwrap(),
+                                    resource_address: state.fungible_resource.unwrap().into(),
```

Comment:
This shouldn't need an `.into()` because we use `AccountSetResourcePreferenceInput` rather than `AccountSetResourcePreferenceManifestInput`.

Reply:
SIDENOTE: In fact this suggests we should really be removing `ManifestSbor` from all the non-manifest models, to avoid this confusion. Then we would need to use `AccountSetResourcePreferenceManifestInput` here...

But happy to delay this until when we have the `HasManifestModel` trait...